src/Umbraco.Core/HealthChecks/Checks/Security/HttpsCheck.cs

```diff
@@ -46,16 +46,16 @@ public class HttpsCheck : HealthCheck
         _hostingEnvironment = hostingEnvironment;
     }
     /// <inheritdoc />
-    public override async Task<IEnumerable<HealthCheckStatus>> GetStatus() =>
-        await Task.WhenAll(
-            CheckIfCurrentSchemeIsHttps(),
-            CheckHttpsConfigurationSetting(),
-            CheckForValidCertificate());
+    public override async Task<IEnumerable<HealthCheckStatus>> GetStatus()
+        => [
+            await CheckIfCurrentSchemeIsHttps(),
+            await CheckHttpsConfigurationSetting(),
+            await CheckForValidCertificate()
+        ];
```

Comment:
This previously started all tasks at the same time, but uses the `ILocalizedTextService` to lookup translations (and any Umbraco specific code, especially services, can create scopes). This could be rewritten to do the actual work concurrently and get nicely formatted health check status messages afterwards, but that might be something for later/another PR.